src/services/applicationService.ts

```diff
@@ -10,4 +11,8 @@ export class ApplicationService {
     async getApplications() {
         return await this.connector.axios.get(`/applications`, {baseURL: this.connector.config.baseURL});
     }
+
+    async surveyUserSignup(applicationId: string, surveyPayload: SurveyPayload) {
+        return await this.connector.axios.post(`/applications/${applicationId}/userSignup`, surveyPayload, {baseURL: this.connector.config.baseURL});
```

Comment:
A few things...

- What type does this method return? If it returns something, please add a type annotation. If it returns nothing, please add the `void` type annotation and remove `return` from line 16. (I recognize that the other methods in this file don't have type annotations, that's a task we will have to complete as well)
- This URL doesn't point to an existing API route. If you are creating a new route to be used by this method, there needs to be an accompanying pull request to the API for it.
- On a side note (this comment should really be on the pull request for the API), please make URLs case insensitive. If you have a compound word like "userSignup", separate words with hyphens, like "user-signup".

Reply:
As far as the user-signup thing, that was a lack of definition causing that issue. I've put a new task in TP to resolve this.